radicle-node/src/parser.rs

```diff
@@ -4,16 +4,16 @@ use std::marker::PhantomData;
 use crate::service::message::Message;
 use crate::wire;
 
-/// Message stream decoder.
+/// Parser of Radicle messages from the network stream.
```

Comment:
A "parser" is typically something that works with a grammar or language. Here, it's a binary stream, so I don't think parser makes sense as a term.

Reply:
My assumption was that the type system of P2P messages is a grammar. But I have no preference and happy to use any name & revert this commit.